src/components/item/ItemPreviewCardGroup.vue

```diff
@@ -151,13 +153,17 @@
 
     data() {
       return {
-        orderedItems: this.items
+        draggableCards: []
```

Comment:
Lets rename this as the cards are not always draggable (on all other views besides sets and the related collections card). For example `orderedCards`